pages/opam-using.html

```diff
@@ -19,6 +19,21 @@ <h2>What is opam?</h2>
 <p>Follow the instructions below to install the last stable version of Coq and
 additional packages. The instructions target an opam newcomer.</p>
 
+<h2 id="platform">The Coq platform scripts</h2>
+
+<p>The <a href="https://github.com/coq/platform">Coq platform</a>
+provide interactive scripts that allow installing Coq and a standard
+set of packages through opam without having to learn anything about
+opam.  We recommend that you use these scripts.  If you do, you can
+skip directly to <a href="#coq-packages">Using opam to install Coq
+packages</a> to learn how to extend the package set that you got.</p>
```

Comment:
"extend" meaning "add additional packages" rather than "edit downloaded packages"?  If so, I'd replace "extend" with "add additional packages".

Reply:
You can do both, but I indeed mean the former.

Reply:
Maybe say the former then.  I tend to think that 90% of users will not extend packages.